test-data/unit/check-unions.test

```diff
@@ -463,3 +463,14 @@ if bool():
     reveal_type(x)  # E: Revealed type is 'Union[builtins.int, builtins.str]'
 reveal_type(x)  # E: Revealed type is 'Union[builtins.int, builtins.str]'
 [builtins fixtures/bool.pyi]
+
+[case testAssignAnyToUnionWithAny]
+from typing import Union, Any
+x: Union[int, Any]
+a: Any
+if bool():
+    x = a
+    # TODO: Maybe we should infer Any as the type instead.
+    reveal_type(x)  # E: Revealed type is 'Any'
```

Comment:
Do we need a TODO item also here? I think the TODO item in the test above is enough, since we already have ``Any`` here. Or am I missing something?